Source/NexusForever.WorldServer/Game/Entity/Inventory.cs

```diff
@@ -321,6 +321,13 @@ public void ItemCreate(Item2Entry itemEntry, uint count, ItemUpdateReason reason
             if (itemEntry == null)
                 throw new ArgumentNullException();
 
+            // auto move materials into satchel but not when moving mats to bag from satchel
+            if (reason != ItemUpdateReason.ResourceConversion) 
+            {
+                if (GameTableManager.Instance.TradeskillMaterial.Entries.SingleOrDefault(i => i.Item2IdStatRevolution == itemEntry.Id) != null)
+                    count = player.SupplySatchelManager.AddAmount(new Item(characterId, itemEntry, count, charges), count);
```

Comment:
```suggestion
                    count = count - player.SupplySatchelManager.AddAmount(new Item(characterId, itemEntry, count, charges), count);
```
because `AddAmount` returns how much was added, not the remainder